debugging/book-library/script.js

```diff
@@ -18,15 +18,18 @@ function populateStorage() {
     myLibrary.push(book2);
     render();
   }
+  
 }
 
 const title = document.getElementById("title");
 const author = document.getElementById("author");
 const pages = document.getElementById("pages");
 const check = document.getElementById("check");
+console.log(title.value, author.value, pages.value, check.checked);
 
 //check the right input from forms and if its ok -> add the new book (object in array)
 //via Book function and start render function
+
```

Comment:
I accidentally approve your PR but your form submission validation needs to be fixed. That's why I'm  leaving this comment